### PR TITLE
Simple mob interaction improvements, cult tile patch

### PR DESCRIPTION
**Pull request by blobbernaut**
There are several items that simple mobs can destroy but do not make a sound or animation when they do, such as chairs. This gives animation and sounds to them for the sake of consistency. 

Artificers now make cult tiles by clicking an adjacent tile/space block, and when it destroys a wall, it automatically becomes a cult floor. 

Yes this method of fixing the cult tile is hacky, but as it stands it's "wait until /tg/ coders fix it". I've tested it and it actually makes being an artificer much more enjoyable and smooth to play as, so until either I fix it or /tg/ fixes it, this should do.

### CODE REVIEW

**Comment by Carbonhell** [comment]
do not merge, remove the FILE_DIRs from tgstation.dme

**Comment by Carbonhell** [comment]
Build tab, Preferences for tg station,uncheck Automatically set file_dir

File: tgstation.dme [hunk 1]
```diff
@@ -6,6 +6,77 @@
 
 // BEGIN_FILE_DIR
 #define FILE_DIR .
+#define FILE_DIR "code"
+#define FILE_DIR "code/modules"
```
Comment (Carbonhell) [inline tgstation.dme]: still here,gotta uncheck it then compile again, and don't forget the int file

File: tgstation.dme [hunk 2]
```diff
@@ -5,7 +5,6 @@
 // END_INTERNALS
 
 // BEGIN_FILE_DIR
-#define FILE_DIR .
```
Comment (Carbonhell) [inline tgstation.dme:8]: fix this

**Comment by Carbonhell** [comment]
Should be good to go

**Comment by Carbonhell** [comment]
File dir is back aaaa

**Comment by blobbernaut** [comment]
huh

**Comment by blobbernaut** [comment]
wew

**Comment by Foosili** [comment]
sounds good 2go :+1:

**Comment by danhibiki** [comment]
Good to me

**Pull request merged**